projects/mat-timepicker/src/lib/timepicker.directive.ts

```diff
@@ -378,7 +379,7 @@ export class MatTimepickerDirective
     }
   }
 
-  @HostListener('keydown', ['$event']) keydownHandler(event: any) {
+  @HostListener('keydown', ['$event']) keydownHandler({ event }: { event: any; }): true | undefined {
```

Comment:
```suggestion
  @HostListener('keydown', ['$event']) keydownHandler(event :  KeyboardEvent): true | undefined {
```
Angular does not wrap then `$event` in an object.